frontend/packages/dev-console/src/components/import/app/ApplicationSelector.tsx

```diff
@@ -34,16 +39,15 @@ const ApplicationSelector: React.FC<ApplicationSelectorProps> = ({ namespace })
 
   const handleOnLoad = (items: { [key: string]: string }) => {
     const isEmptyItems = _.isEmpty(items);
-    setIsZeroApplication(isEmptyItems);
+    setShowAppDropdown(!isEmptyItems);
     if (isEmptyItems) {
       setFieldValue('application.selectedKey', CREATE_APPLICATION_KEY);
       setFieldValue('application.name', '');
     }
   };
-
   return (
     <React.Fragment>
-      {!isZeroApplication && (
+      {showAppDropdown && (
```

Comment:
```suggestion
      {applicationsAvailable && projectsAvailable && (
```
I find this is a lot easier to read when it's explicit: `when applications are available and projects are available, then render the elements`.